packages/@sfcc-bff/basketapi/src/api/schema/basketTypeDef.ts

```diff
@@ -10,6 +10,7 @@ const { gql } = apolloServerCore;
 export const basketTypeDef = gql`
     extend type Query {
         getBasket: Basket!
+        getBasketProductCount: Int!
```

Comment:
Is there any benefit in having a separate resolver for this call? cant we just query getBasket for just one value?

Reply:
getBasket resolver returns a much bigger object (after doing a lot of processing) including product/variation/inventory/shipping etc. This getBasketProductCount is a lightweight call that just returns the number of products in basket that does not require the extra processing.